7.application.md

```diff
@@ -71,76 +92,65 @@ The scopeRef section will index a scope instance by `apiVersion`, `kind` and `na
 
 > The name field is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
 
-#### Trait
-
-The trait section defines an instance of a trait.
-
-| Attribute | Type | Required | Default Value | Description |
-|-----------|------|----------|---------------|-------------|
-| `name`| string | N | |  The name of trait definition. This is used to reference to the definition/schema of the Trait. For one type of trait, there could be only one config/deploy in one component. |
-| `properties`| [`Properties`](#properties) | N | |  The configurable properties that are exposed by this trait. |
-
-> The `name` field must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
-
-
-##### Properties
-
-A properties object for trait is an object whose structure is determined by the [definition reference](6.traits.md#DefinitionRef) of corresponding `TraitDefinition`. It may be a simple value, or it may be a complex object. Properties are validated against the schema appropriate for the trait.
-
 ## Example
 
-The following is an example of a complete YAML file that expresses components and traits. This example illustrates the four configurational elements of a component: its name, workload settings, traits, and scopes.
+The following is an example of a complete YAML file that expresses components, traits, and scopes. This example illustrates the four configurational elements of a component: its name, properties, traits, and scopes.
 
 ```yaml
-apiVersion: core.oam.dev/v1alpha2
+apiVersion: core.oam.dev/v1beta1
 kind: Application
 metadata:
-  name: my-app-deployment
+  name: my-example-app
   annotations:
     version: v1.0.0
-    description: "Description of this deployment"
+    description: "Brief description of the application"
 spec:
   components:
-    - name: my-web-app-component
-      type: Server
-      settings:
-        containers:
-        - name: my-cool-workload
-          image: example/very-cool-workload:0.1.2@sha256:verytrustworthyhash
-          resources:
-            cpu:
-              required: 1.0
-            memory:
-              required: 100MB
-          cmd:
-          - "bash lscpu"
-          ports:
-          - name: http
-            value: 8080
+    - name: publicweb
+      type: web-ui
+      properties: # properties targeting component parameters.
+        image: example/web-ui:v1.0.2@sha256:verytrustworthyhash
+        param_1: "enabled" # param_1 is defined on the web-ui component
       traits:
-        - name: scaler
+        - name: ingress # ingress trait providing a public endpoint for the publicweb component of the application.
```

Comment:
Maybe leave a comment here indicating that the name of the trait will be the same as the component?